src/component/escapable.js

```diff
@@ -0,0 +1,33 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Escapable extends Component {
+  constructor(props) {
+    super(props);
+    this.handleKeyDown = this.handleKeyDown.bind(this);
+  }
+
+  componentDidMount() {
+    if (typeof document !== 'undefined') {
```

Comment:
When checking for a global like `document` or `window` always use `typeof` otherwise you'll get this error in an environment that does not have them: `ReferenceError: document is not defined`